GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -740,8 +739,7 @@ protected override void OnLayout(LayoutEventArgs e)
 
         private void RichTextBox_KeyDown(object sender, KeyEventArgs e)
         {
-            var rtb = sender as RichTextBox;
-            if (rtb is null || !e.Control || e.KeyCode != Keys.C)
+            if (sender is not RichTextBox rtb || !e.Control || e.KeyCode != Keys.C)
```

Comment:
I guess that the pattern matching declaration style is something I have to get used to. In many situations it is probly better too.
I do not like declarations in is not checks though.

Reply:
> I do not like declarations in is not checks though.

It took me a while to get used to it. It's clearer than the alternative, in my opinion, such as:

```c#
if (!(sender is RichTextBox rtb) || ...)
```

Reply:
How about this?
```
if (!e.Control || e.KeyCode != Keys.C || sender is not RichTextBox rtb)
```

Reply:
that order is better for my brain, it is kind of the last result that survives the if()

Reply:
@drewnoakes can you please reorder (few places), then it is good to merge.
Thanks